### ISSUE TITLE
A11y_code.visualstudio.com_DocsLanguages_ Java_Keyboard: The video content plays automatically when user navigates to it.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2779]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: https://code.visualstudio.com
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 3.2.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java)
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
2.	Navigate to "Docs" link.
3.	From left navigation, select "Languages".
4.	Navigate to "Java" link and activate it.
5.	Navigate to the video content available in the main content and observe the video.

**Actual:**
When users navigates to the video content present in the main content area, it start to play automatically and keeps on playing in a loop which is inappropriate.

Similar issue was observed in Edge as well as in Chrome.

**Expected:**
The video content should not start playing automatically instead the video should be played only after the user plays the video on interaction with the video controls.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the video content plays automatically as soon as the user navigates to it then the users with learning impairment might problem in grasping the information and understand the content.

**MAS Reference:**
MAS 3.2.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2})

**Attachment for Reference:**
[Code.visualstudio.com_Docs Setup_ Java_Keyboard Video plays automatically.pptx](https://github.com/microsoft/vscode-docs/files/3314098/Code.visualstudio.com_Docs.Setup_.Java_Keyboard.Video.plays.automatically.pptx)

**Comment by octref** [comment]
Video autoplaying does not constitute as "change of context". This is the default behavior of gif and autoplaying videos.
